examples/ct/ctmapper/mapper/mapper.go

```diff
@@ -36,7 +36,7 @@ import (
 	ct "github.com/google/certificate-transparency-go"
 )
 
-var sourceLog = flag.String("source", "https://ct.googleapis.com/submariner", "Source CT Log")
+var sourceLog = flag.String("source", "https://ct.googleapis.com/submariner", "URL of the CT Log")
```

Comment:
should that still say "source CT log"?

Reply:
There's no "destination" CT log, and the flag itself is called "source", so it seemed superfluous to have that in the description too.

Reply:
OK.